CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog (Pypi package)
 
-## Unreleased
+### [4.9.3] 2023-10-20
```

Comment:
Noticed that the whole file is incorrectly formatted (version title should be h2 and fixed, changed, etc. should be h3) but that's not the scope of this PR !